tests/everflow/conftest.py

```diff
@@ -0,0 +1,35 @@
+import pytest
+import logging
+
+
+@pytest.fixture(autouse=True, scope="module")
+def setup_recycle_port(duthosts, tbinfo):
+    """Setup recycle port ip address on t2 topo"""
+    rec_intf = {}
+    if "t2" in tbinfo['topo']['name']:
+        for duthost in duthosts.frontend_nodes:
+            rec_intf[duthost.hostname] = {}
+            for asic in duthost.asics:
+                output = duthost.command("show ip interfaces -n {} -d all".format(asic.namespace))['stdout_lines']
+                if 'Ethernet-Rec' not in output:
+                    rec_intf[duthost.hostname][asic.namespace] = 1
+                    cmd = "sudo config interface -n {ns} ip add Ethernet-Rec{rec} 1.1.1.{an}/32".format(
```

Comment:
why do we need configure ip address on the Recirc port?

Reply:
It won't route the mirror packets if it doesn't have IP address assigned.  It is described in section 2.3 here https://github.com/sonic-net/SONiC/blob/master/doc/voq/everflow.md